### DISCUSSION TITLE
Dev Review

### DISCUSSION THREAD

**Discussion by jessiejs** [Darth-Ness/WarpCore#36] [General]
Let’s spend a week testing the dev branch, and if there’s an issue, fix it, then reset the one week timer.

**Comment by Darth-Ness** [comment]
Before we merge into beta?

**Comment by jessiejs** [comment]
Yes.

**Comment by Darth-Ness** [comment]
Sounds good to me.

**Comment by Darth-Ness** [comment]
Okay I found a bug, (I also fixed it here [dea1a83](https://github.com/Darth-Ness/WarpCore/commit/dea1a837fe2f2c6f528101c0a273668a2331ec70)) So we reset the timer now.

**Comment by Darth-Ness** [comment]
Should we also wait one week before merging beta into master?
Or should we wait longer?

**Comment by jessiejs** [comment]
We should have 1 week with no bugs found before push.

**Comment by Darth-Ness** [comment]
Okay, so wait a week before pushing dev to beta ,and beta to master?

**Comment by jessiejs** [comment]
We should problably scrap this, because if we make updates in intervals smaller than a week, we end up having to reset the review. Let's just have everyone review the source code once, and if no bugs are found we push.

**Comment by Darth-Ness** [comment]
Good idea. I haven't seen any bugs since that one I fixed.

**Comment by Darth-Ness** [comment]
Have you found any issues?
